subhub/api/webhooks/stripe/invoices.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import json
+
+from subhub.api.webhooks.stripe.abstract import AbstractStripeWebhookEvent
+from subhub.api.webhooks.routes.static import StaticRoutes
+
+logger = logging.getLogger("invoice")
+log_handle = logging.StreamHandler()
+log_handle.setLevel(logging.INFO)
+logformat = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+log_handle.setFormatter(logformat)
+logger.addHandler(log_handle)
+
+
+class StripeInvoiceFinalized(AbstractStripeWebhookEvent):
+    def run(self):
+        d = self.payload
+        sfd = {}
+
+        sfd["event_id"] = d["id"]
+        sfd["event_type"] = d["type"]
+        sfd["customer_id"] = d["data"]["object"]["customer"]
+        sfd["created"] = d["data"]["object"]["created"]
+        sfd["subscription_id"] = d["data"]["object"]["subscription"]
+        sfd["period_end"] = d["data"]["object"]["period_end"]
+        sfd["period_start"] = d["data"]["object"]["period_start"]
+        sfd["amount_paid"] = d["data"]["object"]["amount_paid"]
+        sfd["currency"] = d["data"]["object"]["currency"]
+        sfd["charge"] = d["data"]["object"]["charge"]
+        sfd["invoice_number"] = d["data"]["object"]["number"]
+        sfd["description"] = d["data"]["object"]["lines"]["data"][0]["description"]
+        sfd["application_fee_amount"] = d["data"]["object"]["application_fee_amount"]
+
+        routes = [
+            StaticRoutes.SALESFORCE_ROUTE
+        ]  # setup not complete StaticRoutes.FIREFOX_ROUTE,
```

Comment:
? this comment ?

Reply:
Good catch, I had eyed that yesterday.  I'll pull it and push.